### PR TITLE
Development

### PR DESCRIPTION
**Pull request by omkarborude**

### CODE REVIEW

**Review by kunwar-11 — APPROVED** [review]
Looks good just try to use CSS variables for colors

**Review by deepak-07 — COMMENTED** [review]
Hi Omkar!
You might decrease the box-shadow of options-top as it coming over the header menu

File: index.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.main-info p{
+    font-size: 25px;
+}
+
+.btn-Explore{
+    width: 130px;
+    height: 60px;
+    border: none;
+    font-size: 20px;
+    color: black;
```
Comment (deepak-07) [inline index.css:26]: `outline: none`
You may add this here as well.

**Review by deepak-07 — APPROVED** [review]
Good Job @omkarborude .
You just misspelled badge.css file name, rest it's great.

File: index.css (hunk 1 above)
Comment (omkarborude) [reply index.css:26]: Solved !

File: index.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    font-size: 80px;
+    font-weight: bold;
+    color: orangered;
+}
+
+.main-info p{
+    font-size: 25px;
+}
+
+.btn-Explore{
```
Comment (RohitDhatrak) [inline index.css:21]: try to stay consistent with your naming conventions. (btn-explore)

File: index.css (hunk 1 above)
Comment (RohitDhatrak) [reply index.css:26]: You should not remove the outline it's there for accessibility reasons.

**Review by RohitDhatrak — COMMENTED** [review]
Hey, Omkar good job. I have suggested a few changes.

File: index.css (hunk 2 above)
Comment (omkarborude) [reply index.css:21]: yes!! Thanku 
i just read about outline : none , I will avoid it from now!

**Pull request merged**